setup/install/providers/cf.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: single
+title:  "Cloud Foundry"
+sidebar:
+  nav: setup
+redirect_from: /setup/providers/cf/
+---
+
+{% include alpha version="1.10 and later" %}
+
+{% include toc %}
+
+In [Cloud Foundry](https://www.cloudfoundry.org) (CF), an Account maps to a user account on a CF foundation. You can add multiple accounts for one or more CF foundations.
```

Comment:
@fifthposition 
Searching the CF docs, I can't find CF foundation other than _the_ Cloud Foundry Foundation&mdash;the holding organization of Cloud Foundry. The way you've written this sounds like a CF foundation is an part of the CF object model.